test/studies/ml/lib/Chai.chpl

```diff
@@ -137,32 +137,32 @@ module Chai {
 
         proc init() { }
 
-        proc forwardProp(x: Tensor(?rank)): Tensor(rank) {
+        proc ref forwardProp(x: Tensor(?rank)): Tensor(rank) {
             return tn.sigmoid(x);
         }
 
-        proc forwardPropBatch(batch: [] ?tensorType) where isSubtype(tensorType, Tensor) {
-            return tn.sigmoid(batch);
+        proc ref forwardPropBatch(batch: [] ?tensorType) where isSubtype(tensorType, Tensor) {
+            return forwardProp(batch);
```

Comment:
You also changed the function you called here. Is that OK?

Reply:
Yes, I was getting new promotion errors.